<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>801</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="home">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>150</y>
        <width>96</width>
        <height>60</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu Sans'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; color:#c64600;&quot;&gt;PoLÂ²igono&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="painter">
     <widget class="QPushButton" name="button_apagar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>80</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Apagar</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>23</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton {
    background-color:orange;
    color: white;
    border-radius: 5px;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>23</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton {
    background-color: yellow;
    color: white;
    border-radius: 5px;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>23</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton {
    background-color: blue;
 
    border-radius: 5px;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton {
    background-color: red;
    color: white;
    border-radius: 5px;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>23</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton {
    background-color: green;
    color: white;
    border-radius: 5px;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>23</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton {
    background-color: black;
    color: white;
    border-radius: 5px;
    font-weight: bold;
}
</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="myFrame">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>681</width>
        <height>341</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>800</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu Sans'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Cores&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
